fiftyone/operators/builtins/panels/model_evaluation/__init__.py

```
@@ -0,0 +1,2620 @@
+        )
+        fn_total = (
+            sum(ctx.dataset.values(fn_key))
+            if dataset.has_field(fn_key)
+            else None
+        )
+        return tp_total, fp_total, fn_total
+
+    def get_map(self, results):
+        try:
+            return results.mAP()
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)